### PR TITLE
add to WIP PR

### PR DESCRIPTION
**Pull request by dimalo**
cover basically everything:

- new devices
- most, if not all review triggered changes (except one point)
- multi gateway support
- bugfixes
- first values are instantly fetched instead of waiting for first heartbeat for voltage for example

Some channels are different now, so mind deleting and readding the devices again

### CODE REVIEW

**Comment by kubawolanin** [comment]
Many thanks @dimalo :beer:

File: features/openhab-addons/src/main/feature/feature.xml [hunk 1]
```diff
@@ -17,17 +17,26 @@
...
         <feature>openhab-runtime-base</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.allplay/${project.version}</bundle>
     </feature>
-    
+
     <feature name="openhab-binding-amazondashbutton" description="Amazon Dash Button Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.amazondashbutton/${project.version}</bundle>
     </feature>
+<<<<<<< HEAD
```
Comment (kubawolanin) [inline features/openhab-addons/src/main/feature/feature.xml:30]: @dimalo oops. Seems that there's a conflict here we skipped.

Comment (kubawolanin) [reply features/openhab-addons/src/main/feature/feature.xml:30]: Already resolved :+1:

Comment (dimalo) [reply features/openhab-addons/src/main/feature/feature.xml:30]: hey cool 👍

**Comment by dimalo** [comment]
please test it a little bit first. everyhting should be working as expected.
only the battery items need a bit of reordering...

**Pull request merged**